SpinCore_pp/ppg/echo.py

```diff
@@ -19,24 +19,16 @@
 
 # {{{spin echo ppg
 def run_spin_echo(
-    nScans,
+    settings,
```

Comment:
as noted below, I think the strategy introduced here is problematic.
I think the easiest solution is to roll back to just passing the pulse length to this function.
Because that's what we do for the generic (which we expect to use more, anyways), I think that makes the most sense.

Reply:
I rolled back this edit to the pulse_converting function as well to just take amplitude and deblank_us flat out rather than taking the config_dict as an argument